codegens/curl/lib/index.js

```diff
@@ -4,6 +4,40 @@ var sanitize = require('./util').sanitize,
   _ = require('./lodash'),
   self;
 
+/**
+ *
+ * @param {*} urlObject The request sdk request.url object
+ * @returns {String} final url string converted from parsing url object
+ */
+function getUrlStringfromUrlObject (urlObject) {
```

Comment:
@shreys7 This needs extensive unit tests

Reply:
but this is the implementation that is taken from underyling postman-collection sdk. Do we need to add tests for that too ? @abhijitkane